easy-random-core/src/main/java/org/jeasy/random/util/ReflectionUtils.java

```diff
@@ -133,6 +137,27 @@ public Object invoke(final Object proxy, final Method method, final Object[] arg
      * @throws IllegalAccessException if the property cannot be set
      */
     public static void setProperty(final Object object, final Field field, final Object value) throws IllegalAccessException, InvocationTargetException {
+        //TODO it seems that 'propertyUtilsBean.setProperty(..)' does not work for Maps
+        if(object instanceof java.util.Map){
+            setPropertyForMap(object, field, value);
+        }else{
+            try {
+                final PropertyUtilsBean propertyUtilsBean = new PropertyUtilsBean();
```

Comment:
No need to create a new instance for every call to `setProperty`. I will fix that on merge.